docs/creating_integration_manifest.md

```diff
@@ -116,6 +116,16 @@ Specify the `config_flow` key if your integration has a config flow to create a
 }
 ```
 
+### Single config entry only
+
+Specify the `single_config_entry` key if your integration supports only one config entry. When specified, the frontend will not allow the user to add more than one config entry for this integration.
```

Comment:
```suggestion
Specify the `single_config_entry` key if your integration supports only one config entry. When specified, it will not allow the user to add more than one config entry for this integration.
```